### DISCUSSION TITLE
FR: Specify permissions when creating a section

### DISCUSSION THREAD

**Discussion by chasegiunta** [craftcms/cms#8740] [Ideas]
### Description
We often run into an embarrassing oversight when we add new sections to a site, and that is: we forget to give users/user groups access to edit the section after creation. This results in a email or call from clients wondering where the work is in the control panel that we just finished!

In an effort to streamline workflow, it'd be **_incredible_** if we could specify permissions for the section on the "Create a new section" page itself.

**Comment by lindseydiloreto** [comment]
Yes please! This has happened to me sooo many times.

**Comment by dennisfrank** [comment]
It happened to me more than a few times too.

**Comment by j-greig** [comment]
We have the same problem with globals!
